aptiche/lib/widgets/HomeScreen/drawer.dart

```diff
@@ -33,25 +33,6 @@ class MainDrawer extends StatelessWidget {
                 softWrap: true,
                 style: Theme.of(context).primaryTextTheme.headline2,
               ),
-              /* leading: SizedBox(
```

Comment:
Don't remove this. It's a temporary profile image generator, we have to find a substitute for [this](https://pub.dev/packages/text_drawable_flutter). 
Try using [flutter_text_drawable: ^1.0.0](https://pub.dev/packages/flutter_text_drawable)

Reply:
What? No more apna package? 😢

Reply:
Mil gaya null safe wala.
Apna koi unique wala banayenge

Reply:
merge ho gaya pr. Mujhe abhi show kiya😒

Reply:
![Peace](https://images.bhaskarassets.com/web2images/521/2020/10/06/yo11_1601975331.gif)

Peace 🕊🕊